api/operators-api.ts

```diff
@@ -0,0 +1,1011 @@
+                query.set(key, localVarQueryParameter[key]);
+            }
+            for (const key in options.query) {
+                query.set(key, options.query[key]);
+            }
+            localVarUrlObj.search = (new URLSearchParams(query)).toString();
+            let headersFromBaseOptions = baseOptions && baseOptions.headers ? baseOptions.headers : {};
+            localVarRequestOptions.headers = {...localVarHeaderParameter, ...headersFromBaseOptions, ...options.headers};
+
+            return {
+                url: localVarUrlObj.pathname + localVarUrlObj.search + localVarUrlObj.hash,
+                options: localVarRequestOptions,
+            };
+        },
+        /**
+         * 
+         * @summary List operators
+         * @param {number} [page] Page number starting from 1
+         * @param {number} [perPage] Number of items per page
+         * @param {Array<string>} [name] The account name
+         * @param {Array<string>} [owner] Owner of the project
+         * @param {boolean} [_public] Boolean check for public/private projects
+         * @param {Array<string>} [keyword] A keyword to index the repository by
+         * @param {*} [options] Override http request option.
+         * @throws {RequiredError}
+         */
+        listOperators: async (page?: number, perPage?: number, name?: Array<string>, owner?: Array<string>, _public?: boolean, keyword?: Array<string>, options: any = {}): Promise<RequestArgs> => {
+            const localVarPath = `/operators`;
+            // use dummy base URL string because the URL constructor only accepts absolute URLs.
+            const localVarUrlObj = new URL(localVarPath, 'https://example.com');
+            let baseOptions;
+            if (configuration) {
+                baseOptions = configuration.baseOptions;
+            }
+            const localVarRequestOptions = { method: 'GET', ...baseOptions, ...options};
+            const localVarHeaderParameter = {} as any;
+            const localVarQueryParameter = {} as any;
+
+            // authentication OptionalAuth required
+            // http bearer authentication required
+            if (configuration && configuration.accessToken) {
+                const accessToken = typeof configuration.accessToken === 'function'
+                    ? await configuration.accessToken()
+                    : await configuration.accessToken;
+                localVarHeaderParameter["Authorization"] = "Bearer " + accessToken;
+            }
+
+            if (page !== undefined) {
+                localVarQueryParameter['page'] = page;
+            }
+
+            if (perPage !== undefined) {
+                localVarQueryParameter['per-page'] = perPage;
+            }
+
+            if (name) {
+                localVarQueryParameter['name'] = name;
+            }
+
+            if (owner) {
+                localVarQueryParameter['owner'] = owner;
+            }
+
+            if (_public !== undefined) {
```

Comment:
It looks like this version is properly translating javascript aliases to what the api expects?